src/agent/agent.js

```diff
@@ -198,7 +198,7 @@ class Agent {
         );
 
         // register rpc n2n
-        this.n2n_agent = this.rpc.register_n2n_agent(this.client.node.n2n_signal);
+        this.n2n_agent = this.rpc.register_n2n_agent((...args) => this.client.node.n2n_signal(...args));
```

Comment:
We also have `n2n_proxy` being used in a similar way in server_rpc.
I think you need to fix it too, right?
Can we somehow make sure we didn't miss any other rpc functions passed to functions instead of called from their object?

Reply:
Sadly I have no automatic check I can run to make sure. only the tests we run for each build.
I am reviewing the failed test in order to find other locations

Reply:
All tests have passed so I can only assume the in the coveraged code we do not have these kinds of problems.